tests/test-sources/plugins/utils/which-key.nix

```diff
@@ -251,6 +251,31 @@
               desc = "Vim cmd KeyMapping Test";
               inherit mode;
             }
+            # Nested mapping
```

Comment:
Is it worth porting over the [test we used to have](https://github.com/nix-community/nixvim/blob/62f32bfc711f0a3c8a52af4c0790345b4b3d2346/tests/test-sources/plugins/utils/which-key.nix)? Or do you think this is enough of a test?